src/orange/OrangeParams.cc

```diff
@@ -83,6 +73,40 @@ OrangeInput input_from_json(std::string filename)
     return result;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Load a geometry from the given filename.
+ */
+OrangeInput input_from_file(std::string filename)
+{
+    if (ends_with(filename, ".gdml"))
+    {
+        if (CELERITAS_USE_GEANT4)
+        {
+            // Load with Geant4: must *not* be using run manager
+            auto* world = ::celeritas::load_geant_geometry_native(filename);
+            auto result = g4org::Converter{}(world).input;
+            ::celeritas::reset_geant_geometry();
+            return result;
+        }
+        else
+        {
+            CELER_LOG(warning) << "Using ORANGE geometry with GDML suffix "
```

Comment:
This hack has been the only way we've tested ORANGE vs Geant4 up to now. I don't yet want to remove it because not all the CI (namely, the "fast" ones) could test *any* geometry-related code if we did. Most of the "test" GDML files in the code base, and a couple of the "regression" files in `celer-regression`, do indeed have `x.org.json` alongside them.